hw2/04-background.html

```diff
@@ -6,23 +6,33 @@
     <title>04 - Background</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
   </head>
   <body>
-    <main>
+    <main
+      class="container p-3 bg-light mt-5 text-dark"
+      style="width: 50%; border-radius: 5px"
+    >
       <h1>Exercise 04 - Background</h1>
       <p>
         The background color will change every few seconds (as indicated below).
         You can stop this behavior and you can also change how often the
         background color changes.
       </p>
       <div>
-        <label>Background change interval (seconds): </label>
-        <input />
+        <label for="selector">Background change interval (seconds): </label>
+        <br>
+        <select class="custom-select" id="selector" style="width: 15em;">
```

Comment:
One small suggestion, I think we are not supposed to use dropdown for the time input. As the professor mentioned in the exercise description when no time is mentioned background color should change for every three seconds. And also it does not match the screenshot she provided.